purchase_supplier_approve/models/res_partner.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2019 RGB Consulting - Domantas Sidorenkovas
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    supplier_approval = fields.Selection([
+        ('pda', 'Pending approval'),
+        ('apd', 'Approved'),
+        ('nad', 'Not approved'),
+    ], default='pda', readonly=True, track_visibility='onchange')
```

Comment:
It would be great if you can add help string.

Reply:
Hi again,
As you asked, I made this change also in the following commit:
https://github.com/OCA/purchase-workflow/pull/744/commits/b6f823d3cfa9d4a71fd8cc4b6128cda08aeb6d33#diff-d44cb370023ae01225035b3421763d4eR15

Reply:
Looks good :+1: